op-e2e/actions/helpers/user.go

```diff
@@ -444,7 +447,7 @@ func (s *CrossLayerUser) getLatestWithdrawalParams(t Testing) (*withdrawals.Prov
 		return nil, fmt.Errorf("the latest L2 output is %d and is not past L2 block %d that includes the withdrawal yet, no withdrawal can be proved yet", l2OutputBlock.NumberU64(), l2WithdrawalBlock.NumberU64())
 	}
 
-	if !e2eutils.UseFaultProofs() {
+	if s.allocType == config.AllocTypeL2OO {
```

Comment:
```suggestion
	if !s.allocType.SupportsProofs() {
```

And then we should update `SupportsProofs` to include alt-da. It's only the L2OO variant that doesn't use proofs for withdrawals.  https://github.com/ethereum-optimism/optimism/blob/da9bc723d83570f3e8e1a4388228588bba8112d2/bedrock-devnet/devnet/__init__.py#L126-L127